CHANGES.rst

```diff
@@ -6,6 +6,13 @@ CHANGES
 2.0.2 (unreleased)
 ------------------
 
+- Backwards incompatible change: PJDataManager now accepts a pool instead
```

Comment:
Feels like this should be a major version then?

Reply:
+1

Reply:
3.0.0 is pending for release